charts/jindofsx/values.yaml

```diff
@@ -35,6 +35,7 @@ master:
   dnsServer: ""
   labels: {}
   annotations: {}
+  env: {}
```

Comment:
Please also update chart version and changelog. thanks.

Reply:
Done